src/Features/Core/Portable/Completion/CompletionHelperServiceFactory.cs

```diff
@@ -1,81 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System;
-using System.Composition;
-using Microsoft.CodeAnalysis.Host;
-using Microsoft.CodeAnalysis.Host.Mef;
-using Microsoft.CodeAnalysis.LanguageService;
-using Microsoft.CodeAnalysis.Shared.Extensions;
-using Roslyn.Utilities;
-
-namespace Microsoft.CodeAnalysis.Completion
-{
-    [ExportWorkspaceServiceFactory(typeof(ICompletionHelperService)), Shared]
-    internal class CompletionHelperServiceFactory : IWorkspaceServiceFactory
-    {
-        [ImportingConstructor]
-        [Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
-        public CompletionHelperServiceFactory()
-        {
-        }
-
-        public IWorkspaceService CreateService(HostWorkspaceServices workspaceServices)
-            => new Service(workspaceServices.Workspace);
-
-        private sealed class Service : ICompletionHelperService, IWorkspaceService
-        {
-            private readonly object _gate = new();
-
-            private CompletionHelper? _lazyCaseSensitiveInstance;
```

Comment:
Singleton now moved into CompletionHelper itself. So this service is no longer needed